data/coalition/coalition missions.txt

```diff
@@ -1866,3 +1866,469 @@ mission "Saryd University Lecture"
+			`	You two look at the pylon in silence for a while, then she slowly approaches the machine to pick it up, saying "Supposed to head back home immediately, expedition ships were, if destroyed, one of the Punishers was. If followed those orders, he had... No guarantee of the split group's survival, he had. If back home, they went..."`
+			choice
+				`	"He could have, but then the other group wouldn't know that any Punisher was destroyed. They wouldn't know to go back."`
+				`	"Seems like they didn't pick the best person to follow those orders."`
+				`	"We don't know if he could've made it if he had gone back immediately. I'm sure we'll find out more with the next pylon."`
+					goto more
+			`	She doesn't respond, instead looking down, at the lines carving the clearing's ground. She moves close to one, laying down and reaching her arms to touch the cracks. A few minutes later, she gets back up, and looks to you.`
+			label more
+			`	"Back to the ship, we should go," she says. "Try and figure out where Mimodum went next, I will. Where he was to meet with the split group."`
+			`	After taking the trail back, Sedlitaris goes into her bunk to tinker with the pylons. She comes out hours after nightfall, asking that you show her your map. "A pincer movement, the two groups were performing. Difficult to know the exact planet, it is, as the data from the split group, we don't have, but figured the overall region where the two groups were link up, I think I have."`
+			`	You help her match the coordinates, and end up with a group of systems between Sol and Tarazed. "Need to visit each system, we might. Pick up faint signals of their next pylon, while in the right system I can, but no way of knowing where that is exactly, I have."`
+				accept
+	on enter "Rasalhague"
+		dialog `Sedlitaris has the pylon start searching for any matching signals as soon as you jump into the system. A few hours pass, and the machine doesn't return any coordinates, however it also doesn't report a failure. Sedlitaris looks at it puzzled, and reboots it. "Fly around the system for a while, could you, Captain? Maybe compromising the signal, the ship's current position is." You do as she suggests, and slowly cruise around the system for a few more hours. Once again, the pylon doesn't give any answer. Sedlitaris says it might be some oddly specific bug, and asks that you just move on to the next system.`
```

Comment:
```suggestion
		dialog `Sedlitaris has the pylon start searching for any matching signals as soon as you jump into the system. A few hours pass and the machine doesn't return any coordinates, but it also doesn't report a failure. Sedlitaris looks at it puzzled, and reboots it. "Fly around the system for a while, could you, Captain? Maybe compromising the signal, the ship's current position is." You do as she suggests, and slowly cruise around the system for a few more hours. Once again, the pylon doesn't give any answer. Sedlitaris says it might be some oddly specific bug, and asks that you just move on to the next system.`
```